src/pages/Account/Venue/VenueMapEdition/Container.tsx

```diff
@@ -70,17 +72,26 @@ export const Container: React.FC<PropsType> = (props) => {
 
   useLayoutEffect(() => {
     mapRef?.current?.getBoundingClientRect().width &&
-      setScale(
-        mapRef.current.getBoundingClientRect().width / PLAYA_WIDTH_AND_HEIGHT
-      );
+      setScale({
```

Comment:
(Edit: this comment was a suggestion about DRYing the code but it’s already done at the setScale level; no action needed here)